docs/strategies/timeout.md

```diff
@@ -119,3 +124,51 @@ sequenceDiagram
     T->>P: Throws <br/>TimeoutRejectedException
     P->>C: Propagates exception
 ```
+
+## Anti-patterns
+
+### Ignoring Cancellation Token
```

Comment:
On other pages the anti-patterns are numbered. I think it might make sense to add numbering here as well 

```suggestion
### 1 - Ignoring Cancellation Token
```

Reply:
Does the number add any value though?

Reply:
If you have multiple paragraphs and one might refer to a previous one then its number could enough.

But I also did not follow this :D
- http://localhost:8080/strategies/retry.html#6---having-a-single-strategy-for-multiple-failures
- It is referring to the 5th paragraph via _Previously, it was suggested that_

Reply:
Let me delete all the numbers in my next PR where I change the `[!IMPORTANT]` blocks to `[!INFO]`